src/legacy/work/Header/Header.tsx

```diff
@@ -106,7 +108,7 @@ const DetailedHeader = () => {
                               {isHost ? 'Host Profile' : 'Become a Host'}
                             </Link>
                             {authNavItems.map(item => (
-                              <NavItem className="px-2" key={item.header}>
+                              <NavItem className={['px-2', item.className].join(' ')} key={item.header}>
```

Comment:
This surprised me a little bit:

    $ node
    > ['a', undefined].join(' ')
    'a '

I'd have expected `'a undefined'`

We tend to do a bit more finessing when `className` comes from multiple places, so figured I'd call this out in case there is something I'm missing.

Reply:
there's a package for that :) https://www.npmjs.com/package/classnames

Reply:
Hehe, yeah, ran across that in [this thread](https://github.com/facebook/react/issues/3138), which is also where the [`join`](https://github.com/facebook/react/issues/3138#issuecomment-74311220) solution came from. Almost went with the former but couldn't justify it when a built-in is functionally equivalent (for this usage).

Only drawback I see with `join` is that the behavior is potentially non-obvious, at the language level:

    ['a', undefined, 'b'].join(' ') !== 'a' + ' ' + undefined + ' ' + 'b'